l10n_br_account_service/models/res_company.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#                                                                             #
+# Copyright (C) 2009  Renato Lima - Akretion                                  #
+#                                                                             #
+# This program is free software: you can redistribute it and/or modify        #
+# it under the terms of the GNU Affero General Public License as published by #
+# the Free Software Foundation, either version 3 of the License, or           #
+# (at your option) any later version.                                         #
+#                                                                             #
+# This program is distributed in the hope that it will be useful,             #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of              #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the               #
+# GNU Affero General Public License for more details.                         #
+#                                                                             #
+# You should have received a copy of the GNU Affero General Public License    #
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.       #
+###############################################################################
+
+from openerp import models, fields, api
+
+from openerp.addons.l10n_br_account.models.l10n_br_account import (
+    L10n_brTaxDefinition
+)
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    @api.one
+    @api.depends('service_tax_definition_line.tax_id')
+    def _compute_service_taxes(self):
+        service_taxes = self.env['account.tax']
+        for tax in self.service_tax_definition_line:
+            service_taxes += tax.tax_id
+        self.product_tax_ids = service_taxes
+
+    service_invoice_id = fields.Many2one(
+        'l10n_br_account.fiscal.document',
+        'Documento Fiscal')
+    document_serie_service_id = fields.Many2one(
+        'l10n_br_account.document.serie', u'Série Fiscais para Serviço',
+        domain="[('company_id', '=', active_id),('active','=',True),"
+        "('fiscal_type','=','service')]")
+    nfse_version = fields.Selection(
+        [('100', '1.00')], 'Versão NFse', required=True, default="100")
+    nfse_import_folder = fields.Char('Pasta de Origem', size=254)
+    nfse_export_folder = fields.Char('Pasta de Destino', size=254)
+    nfse_backup_folder = fields.Char('Pasta de Backup', size=254)
```

Comment:
@danimaribeiro 

Inside eletronic_docs_folder we will need create sub folders to nfe and nfse, we can leave this and think more when we work in nfse.